types/smokehouse.d.ts

```diff
@@ -19,14 +19,28 @@
     diff?: Difference | null;
   }
 
-  export type ExpectedLHR = Pick<LH.Result, 'audits' | 'finalUrl' | 'requestedUrl'> & { errorCode?: string }
+  export type ExpectedLHR = Pick<LH.Result, 'audits' | 'finalUrl' | 'requestedUrl'>
 
-  export interface LHRComparison {
-    audits: Comparison[];
+  export type ExpectedRunResult = {
+    errorCode?: string;
+    lhr: ExpectedLHR,
+    artifacts?: Partial<LH.Artifacts>
+  }
+
+  export interface RunComparison {
+    assertions: Comparison[];
     errorCode: Comparison;
     finalUrl: Comparison;
   }
 
+  export interface Assertion{
```

Comment:
nit: space after `Assertion`

Reply:
Whoops, realized this is the same as `Smokehouse.Comparison`.